### PR TITLE
Using untill/gojay fork

### PR DESCRIPTION
**Pull request by Yohanson555**
Decrease memory usage due to usage of UnmarshalJSONObjectWithPool method from gojay fork

### CODE REVIEW

File: dynostorage.go [hunk 1]
```diff
@@ -155,6 +160,7 @@ func putBufferSlice(b *BuffersSlice) {
 	b.Owner = nil
 
 	BuffersPool.Put(b)
+	//StatIncrement("BuffersSlicePut", 1)
```
Comment (maxim-ge) [inline dynostorage.go:163]: Remove this comment

File: dynostorage.go [hunk 2]
```diff
@@ -31,9 +33,17 @@ var BufferPool = sync.Pool{
 
 func getBuffer() (b *Buffer) {
 	b = BufferPool.Get().(*Buffer)
+	b.isReleased = false
```
Comment (maxim-ge) [inline dynostorage.go:36]: Remove this

**Review by maxim-ge — CHANGES_REQUESTED** [review]

File: dynostorage.go (hunk 2 above)
Comment (Yohanson555) [reply dynostorage.go]: removed

File: dynostorage.go (hunk 1 above)
Comment (Yohanson555) [reply dynostorage.go]: removed

**Pull request merged**